extensions/interactions/NumberWithUnits/directives/number-with-units-validation.service.spec.ts

```diff
@@ -240,7 +240,12 @@ describe('NumberWithUnitsValidationService', () => {
         }
       },
       toMathjsCompatibleString: () => {
-        return null as unknown as string;
+        // This throws "Type null is not assignable to type
+        // 'string'." We need to suppress this error
+        // because of the need to test validations. This
+        // function is not used in the validations.
+        // @ts-ignore
+        return null as string;
```

Comment:
This doesn't look good. Usually we use this to pass a Partial value type to a component for testing. You will need to figure out what is this.